packages/cardhost/tests/acceptance/card-mode-navigation-test.js

```diff
@@ -14,8 +20,18 @@ const testCard = cardDocument().withAutoAttributes({
   title: 'The Millenial Puppy',
 });
 const cardPath = encodeURIComponent(testCard.canonicalURL);
+
+const adoptableTestCard = cardDocument().withAutoAttributes({
+  csRealm,
+  csId: 'gen-z-puppies',
+  title: 'The Gen Z Puppy',
+});
+const adoptableCardPath = encodeURIComponent(adoptableTestCard.canonicalURL);
 const scenario = new Fixtures({
-  create: [testCard],
+  create: [testCard, adoptableTestCard],
+  destroy: {
+    cardTypes: [adoptableTestCard],
+  },
```

Comment:
you actually have a test leak here. Please note that this Fixture is  module scoped (line 38):
```js
  scenario.setupModule(hooks);
```

This is basically a bug lying in wait--you are not returning the system back to a clean state because this Fixture is not being run after your test--it's being run after the module itself. This means that the card that was created in your test is leaking into all the other tests in this module.

This type of test is a "mutating" test, and as such, it cannot live in this module, because this module is setup for "read-only" tests. You can either create a new module in this file specifically for mutating tests like this that uses a Fixture that run at the test boundary. or, what might be easier, would be to move this test into the adoption test suite, which is already setup as "mutating" tests.